stf/feature_engineering/feature_free_days.py

```diff
@@ -43,27 +48,83 @@ def create_holiday_functions():
                 consist of "Is" + the_name_of_the_holiday_to_be_checked
     """
 
-    # Manully generated csv including all dutch holidays for different regions
-    df_holidays = pd.read_csv(HOLIDAY_CSV_PATH, index_col=None)
-    df_holidays["datum"] = pd.to_datetime(df_holidays.datum).apply(lambda x: x.date())
+    if years is None:
+        now = datetime.now()
+        years = [now.year - 1, now.year]
+
+    country_holidays = holidays.CountryHoliday(country, years=years)
 
     # Make holiday function dict
     holiday_functions = {}
     # Add check function that includes all holidays of the provided csv
     holiday_functions.update(
-        {"IsFeestdag": lambda x: np.isin(x.index.date, df_holidays.datum.values)}
+        {"is_national_holiday": lambda x: np.isin(x.index.date, np.array(list(country_holidays)))}
     )
 
     # Loop over list of holidays names
-    for holidayname in list(set(df_holidays.name)):
+    for date, holiday_name in sorted(country_holidays.items()):
         # Define function explicitely to mitigate 'late binding' problem
-        def make_holiday_func(holidayname=holidayname):
+        def make_holiday_func(date):
+            return lambda x: np.isin(x.index.date, np.array([date]))
+
+        # Create lag function for each holiday
+        holiday_functions.update(
+            {"is_" + holiday_name.replace(" ", "_").lower(): make_holiday_func(date)}
+        )
+
+        # Check for bridgedays
+        # Looking forward: If day after tomorow is a national holiday or
+        # a saturday check if tomorow is not a national holiday
+        if (date + timedelta(days=2)) in country_holidays or\
+                (date + timedelta(days=2)).weekday() == 5:
+
+            # If tomorow is not a national holiday or a weekend day make it a bridgeday
+            if not (date + timedelta(days=1)) in country_holidays \
+                    and not (date + timedelta(days=1)).weekday() in [5, 6]:
+
+                # Create feature function for each holiday
+                holiday_functions.update(
+                    {"is_bridgeday" + holiday_name.replace(" ", "_").lower(): make_holiday_func((date + timedelta(days=1)))}
+                )
```

Comment:
Bridgedays: Nice addition 
Can 'bridgeday' without the holidayname be a more significant feature? 

We also have the generic 'is_holiday' feature. Can we add 'is_bridgeday' as well? (or perhaps even say that is_holiday=True for bridgedays)